bundles/org.eclipse.team.ui/src/org/eclipse/team/internal/ui/dialogs/PreferencePageContainerDialog.java

```diff
@@ -159,16 +159,17 @@ protected Control createDialogArea(Composite parent) {
 
 	private void createMultiplePageArea(Composite composite) {
 		// create a tab folder for the page
-		tabFolder = new TabFolder(composite, SWT.NONE);
+		tabFolder = new CTabFolder(composite, SWT.NONE);
 		tabFolder.setLayoutData(new GridData(GridData.FILL_BOTH));
 
 		for (PreferencePage page : pages) {
 			// text decoration options
-			TabItem tabItem = new TabItem(tabFolder, SWT.NONE);
+			CTabItem tabItem = new CTabItem(tabFolder, SWT.NONE);
 			tabItem.setText(page.getTitle());//
 			tabItem.setControl(createPageArea(tabFolder, page));
 			pageMap.put(tabItem, page);
 		}
+		tabFolder.setSelection(0);
```

Comment:
With https://github.com/eclipse-platform/eclipse.platform.swt/issues/46 fixed, this line would not be needed.